dbms/tests/queries/0_stateless/00633_func_or_in.sql

```diff
@@ -0,0 +1,8 @@
+drop table if exists test.orin_test;
```

Comment:
Could you please merge with master
(because these changes are already in master)